docs/started.md

```diff
@@ -54,7 +54,7 @@ See [the rule list](../rules/)
   - Object option
     - `pattern` (`string`) ... A glob for specifying files that store localization messages of project.
     - `localeKey` (`'file' | 'key'`) ... Specifies how to determine the locale for localization messages.
-      - `'file'` ... Determine the locale name from the filename. The resource file should only contain messages for that locale. Use this option if you use `vue-cli-plugin-i18n`. This option is also used when String option is specified.
+      - `'file'` ... Determine the locale name from the filename. The resource file should only contain messages for that locale. Use this option if you use this plugin. This option is also used when String option is specified.
```

Comment:
This means that it is useful to use this option when using it in the initial state generated by `vue-cli-plugin-i18n`, so I don't think it is "this plugin".

https://github.com/kazupon/vue-cli-plugin-i18n/blob/67c9f003fb162f1893b78a32b4ec7a68dfb56a0a/generator/templates/js/src/i18n.js#L13


I don't understand English so I don't think my grammar is correct:sweat_smile:

Reply:
Ah sorry, I was misunderstanding... NVM ...